src/lib/index.ts

```diff
@@ -8,50 +8,41 @@
 
 import * as debug from "debug";
 
-import createStore from "./store";
-import sagas from "./sagas";
 import reducer from "./reducer";
+import sagas from "./sagas";
+import createStore from "./store";
 
-import { IRequest } from "./types";
 import { add } from "./actions";
+import { IRequest } from "./types";
 import numericRandomId from "./utils/numericRandomId";
 
 const dbg: debug.IDebugger = debug("@stylelounge/http-queue");
 
 const middlewares = { sagas };
 
-type HttpQueue = {
-    schedule: (manifest: IRequest) => void,
-    drain: (timeout?: number) => Promise<void>
-};
+interface IHttpQueue {
+    schedule: (manifest: IRequest) => void;
+    drain: (timeout?: number) => Promise<void>;
+}
 
-const createHttpQueue = (): HttpQueue => {
+const createHttpQueue = (forceXHR: boolean): IHttpQueue => {
     const store = createStore({ reducer, middlewares });
 
     /**
      * Schedules a HTTP request.
-     *
-     * @param {IRequest} request The request definition
-     * @returns void
-     *
      */
     const schedule = (manifest: IRequest) => {
         store.dispatch(
             add({
+                forceXHR,
```

Comment:
Should be called `forceXhr`, we do e.g. "HttpProvider" and "id" instead of "HTTPProvider" and "ID"